### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by pvacca97**

### CODE REVIEW

File: SA3/qa_web/templates/qa_web/index.html [hunk 1]
```diff
@@ -14,6 +14,12 @@ <h1>Temporary Index Page for SOEN341 W18 group SA3.</h1>
 <a href="./login/"><button>Login</button></a>
 <a href="./signup/"><button>Signup</button></a>
 
+{% if user.is_authenticated %}
```
Comment (alvyn279) [inline SA3/qa_web/templates/qa_web/index.html]: Good thing the logout button is present only if the user is logged in.

File: SA3/qa_web/views.py [hunk 2]
```diff
@@ -51,6 +51,10 @@ def login(request):
         else:
             return render_to_response('qa_web/login.html', context={'form': form})
 
+def logout_view(request):
+    logout(request)
+    return HttpResponseRedirect('../login/')
+    # redirects to log in page once logged out successfully
```
Comment (alvyn279) [inline SA3/qa_web/views.py]: I believe the simple `logout` function call you implemented here is enough, as we do not need the session data that goes with the user logged in (it usually gets all flushed away with `logout`). All of the user interaction is handled through actual data being saved on the database. Sick redirect too.

File: requirements.txt [hunk 3]
```diff
@@ -1 +1,2 @@
-django==2.0
\ No newline at end of file
+django==2.0
+beautifulsoup4==4.6.0
```
Comment (alvyn279) [inline requirements.txt]: @pvacca97 Why do you need this library?

Comment (alvyn279) [reply requirements.txt]: Like I don't understand why you need to pull data from HTML and XML in this PR

Comment (pvacca97) [reply requirements.txt]: I thought I was going to use this library but that didn't end up being the case... Might still be useful later on when we want to manipulate the homepage welcoming the user signing in?

Comment (Lercerss) [reply requirements.txt]: It might be best to remove it until then and add it later if the need arises.

**Comment by Lercerss** [comment]
Minor changes required before we can merge: rebase the branch and remove the library from requirements as it is not being used right now.

**Pull request merged**